tuf/utils/pkcs8.go

```diff
@@ -0,0 +1,299 @@
+package utils
```

Comment:
Yes. Its plucked from MIT licensed source, should I include the license at the top of the file?